test/common/helm_install.go

```diff
@@ -0,0 +1,62 @@
+// Package common contains utility functions for all the tests in this repo
+package common
```

Comment:
The name is better be something like test_util

Reply:
changed pkg name to test_util